Makefile.am

```diff
@@ -85,3 +86,9 @@ desktopdir = $(datadir)/applications
 desktop_in_files = geany.desktop
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
+
+appdata_in_files = geany.appdata.xml.in
```

Comment:
this should be at the same place as the desktop one relative to the dir and other definitions, and the extra blank line removed.